doc/whats_new.rst

```diff
@@ -7,6 +7,7 @@ NEW
 Fixes
 -----
 
+- Fix plot title overlapping image content.
```

Comment:
Could you add the issue number in the description here? It might be hard to know what was fixed otherwise.